.gitlab/e2e/e2e.yml

```
@@ -54,8 +54,8 @@
     - export DD_CIVISIBILITY_AGENTLESS_ENABLED=true
     - DD_API_KEY=$($CI_PROJECT_DIR/tools/ci/fetch_secret.sh $AGENT_API_KEY_ORG2 token) || exit $?; export DD_API_KEY
     # set the environment variables for the windows test signing
-    - export WINDOWS_DDNPM_DRIVER=$(dda inv release.get-release-json-value "dependencies::WINDOWS_DDNPM_DRIVER" --no-worktree)
-    - export WINDOWS_DDPROCMON_DRIVER=$(dda inv release.get-release-json-value "dependencies::WINDOWS_DDPROCMON_DRIVER" --no-worktree)
+    - if [ -z "${WINDOWS_DDNPM_DRIVER:-}" ]; then export WINDOWS_DDNPM_DRIVER=$(dda inv release.get-release-json-value "dependencies::WINDOWS_DDNPM_DRIVER" --no-worktree); fi
+    - if [ -z "${WINDOWS_DDPROCMON_DRIVER:-}" ]; then export WINDOWS_DDPROCMON_DRIVER=$(dda inv release.get-release-json-value "dependencies::WINDOWS_DDPROCMON_DRIVER" --no-worktree); fi
```

🥜 As you are checking if these variables are empty strings, you don't need to define them to "" if there are not defined I guess.
```suggestion
    - if [ -z "${WINDOWS_DDNPM_DRIVER}" ]; then export WINDOWS_DDNPM_DRIVER=$(dda inv release.get-release-json-value "dependencies::WINDOWS_DDNPM_DRIVER" --no-worktree); fi
    - if [ -z "${WINDOWS_DDPROCMON_DRIVER}" ]; then export WINDOWS_DDPROCMON_DRIVER=$(dda inv release.get-release-json-value "dependencies::WINDOWS_DDPROCMON_DRIVER" --no-worktree); fi
```